lib/common/netcdf-check.sh

```diff
@@ -109,20 +95,17 @@ add_checker_signature() {
     local checker_version=`$NETCDF_CHECKER --version`
     local version_number=`echo $checker_version | egrep 'IOOS compliance checker version' | egrep -o '[0-9.]+$'`
 
-    local commit_hash=`_get_checker_commit_hash`
-    local last_updated=`_get_checker_commit_date`
+    local imos_version=$($NETCDF_CHECKER -l | grep ' imos ' | sed -r 's/.*\((.*)\)/\1/')
```

Comment:
FYI, the Chef recipe tests the versions with the following one-liners... not very pretty but serviceable until able to be queried out of a Python context.

```bash
checker_ver=$(pip show compliance-checker | awk /^Version:/'{print $2}')
plugin_ver=$(pip show cc-plugin-imos | awk /^Version:/'{print $2}')
```